Subject: Increase the time KDM waits for X to start up
From: Lubos Lunak
Patch-upstream: no
Bug: bnc#462478

See the bugreport. In short, some drivers suck.

Index: kdm/config.def
===================================================================
--- kdm/config.def.orig
+++ kdm/config.def
@@ -1363,7 +1363,7 @@ Description:
 
 Key: ServerTimeout
 Type: int
-Default: 30
+Default: 60
 User: core
 Instance: #:*/
 Comment:
